### PR TITLE
REDCORE-570 refactor for test case of InstallExtensionCest

### PR DESCRIPTION
**Pull request by ThuNguyen1411**

### CODE REVIEW

File: tests/_support/Step/Acceptance/ActivateWebServicesSteps.php [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+		$I= $this;
+		$I->amOnPage(ActWebPage::$URL2);
+		$I->waitForText(ActWebPage::$textManager, 30, ActWebPage::$h1);
+		$I->click(ActWebPage::$buttonNotInstall);
+		$I->waitForElement(ActWebPage::$installElement, 30);
+		$I->executeJS("javascript:window.scrollBy(0,200);");
+		$I->click(ActWebPage::$buttonInstall);
+		$I->waitForElement(ActWebPage::$tableForm, 60);
+		$I->waitForText(ActWebPage::$pathContactWebservice1, 30, ActWebPage::returnXpath(ActWebPage::$pathContactWebservice1));
+		$I->waitForText(ActWebPage::$pathContactWebservice2,30 , ActWebPage::returnXpath(ActWebPage::$pathContactWebservice2));
```
Comment (NguyenTrang091296) [inline tests/_support/Step/Acceptance/ActivateWebServicesSteps.php]: add space after `,`

File: tests/_support/Step/Acceptance/AbstractStep.php [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+ * @copyright   Copyright (C) 2008 - 2019 redCOMPONENT.com. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+namespace Step\Acceptance;
+
+/**
+ * Class AbstractStep
+ * @package Step\Acceptance
+ */
+class AbstractStep extends \AcceptanceTester
```
Comment (nhungredweb) [inline tests/_support/Step/Acceptance/AbstractStep.php]: seem like we don't have any value for this class .

**Comment by nhungredweb** [comment]
hi @puneet0191  can you please help us review again . thank you so much.

File: tests/_support/Page/AbstractPage.php [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+		 */
+		public static $h1                     = ['css' => 'h1'];
+
+		/**
+		 * @var string
+		 */
+		public static $buttonSave             = 'Save';
+
+		/**
+		 * @var array
```
Comment (puneet0191) [inline tests/_support/Page/AbstractPage.php]: what spacing have you used here?

File: tests/acceptance/install/02-InstallExtensionCest.php [hunk 4]
```diff
@@ -16,31 +24,15 @@ public function install(\AcceptanceTester $I)
...
-		$I->waitForText('Webservice Manager', 30, ['css' => 'H1']);
-		$I->click(['class' => 'lc-not_installed_webservices']);
-		$I->waitForElement(['class' => 'lc-install_all_webservices'], 60);
-		$I->executeJS("javascript:window.scrollBy(0,200);");
-		$I->click(['class' => 'lc-install_all_webservices']);
-		$I->waitForElement(['id' => 'oauthClientsList'], 30);
-		$I->see('administrator.contact.1.0.0.xml', ['class' => 'lc-webservice-file']);
-		$I->see('site.contact.1.0.0.xml', ['class' => 'lc-webservice-file']);
-		$I->see('site.users.1.0.0.xml', ['class' => 'lc-webservice-file']);
+		$I->doActivateWebservices();
```
Comment (puneet0191) [inline tests/acceptance/install/02-InstallExtensionCest.php]: ActivateWebservices
No need to add a "do", it is not correct English I->do, rather I->activate

**Comment by ThuNguyen1411** [comment]
hi @puneet0191 can you please help me review again . thank you so much.

**Comment by nhungredweb** [comment]
thanks @puneet0191 , @ThuNguyen1411 , @NguyenTrang091296  so much

**Pull request merged**